bbmri/docker-compose.yml

```diff
@@ -8,8 +8,8 @@ services:
     container_name: bridgehead-bbmri-blaze
     environment:
       BASE_URL: "http://bridgehead-bbmri-blaze:8080"
-      JAVA_TOOL_OPTIONS: "-Xmx4g"
-      LOG_LEVEL: "debug"
+      JAVA_TOOL_OPTIONS: "-Xmx${BLAZE_MEMORY_CAP:-4096}m"
+      DB_BLOCK_CACHE_SIZE: $BLAZE_MEMORY_CAP
```

Comment:
DB_RESOURCE_CACHE_SIZE is missing which is also an important Value. We would need for 2500000 for 4GB RAM. However this value depends on the physical size of data and the amount of resources.